oalab/src/openalea/oalab/plugins/controls/widgets.py

```diff
@@ -47,6 +47,166 @@ def value(self, interface=None):
         return self.text()
 
 
+
+class QFloatSlider(QtGui.QSlider):
+
+    floatValueChanged = QtCore.Signal(float)
+
+    def __init__(self,orientation=QtCore.Qt.Horizontal):
+        QtGui.QSlider.__init__(self,orientation)
+        self.connect(self,QtCore.SIGNAL('valueChanged(int)'),self.notifyValueChanged)
+        self.slider_step = 0.1
+        self.floatValue = 0.0
+
+    def notifyValueChanged(self,value):
+        self.floatValue = value*self.slider_step
```

Comment:
We try to use python convention (lower case for class methods and attributes).
This is a bit ambiguous here as Qt widget uses mixedCase convention.
Current choice is:
  - mixedCase: overidden methods and Qt specific methods and attributes like signals, contextMenuEvent, ...
  - lower_case: all other methods and attributes